test/Concurrency/actor_isolation.swift

```diff
@@ -1659,3 +1659,33 @@ actor SafeMutatingCall {
     _ = try await self.iterator.next()
   }
 }
+
+
+@MainActor
+func testLocalFunctoinIsolation() {
```

Comment:
minor: don't let this hold off merging but typo here
```suggestion
func testLocalFunctionIsolation() {
```

Reply:
Whoops, thanks! I'm going to merge this; I've got another diagnostics change locally and I'll fix it along with that.

Reply:
Fixed in https://github.com/swiftlang/swift/pull/75687